test/gateway/eip20_gateway/stake.js

```diff
@@ -156,18 +156,24 @@ contract('EIP20Gateway.stake() ', function (accounts) {
 
   it('should fail to stake when stake amount is 0', async function () {
     stakeAmount = new BN(0);
-    errorMessage = "Stake amount must not be zero";
+    errorMessage = "Stake amount must not be zero.";
     await prepareData();
     await stake(utils.ResultType.FAIL);
   });
 
   it('should fail to stake when beneficiary address is 0', async function () {
     beneficiary = Utils.NULL_ADDRESS;
-    errorMessage = "Beneficiary address must not be zero";
+    errorMessage = "Beneficiary address must not be zero.";
     await prepareData();
     await stake(utils.ResultType.FAIL);
   });
 
+  it('should fail to max reward amount is greater than the stake amount', async function () {
+    stakeAmount = new BN(200);
```

Comment:
What's the max reward amount here? Seems a little opaque.

Reply:
Added comment in the file
```
    /*
     * Max reward amount will be `gasPrice * gasLimit`.
     * i.e. in this case its 180000000, which is higher than the stake amount.
     */
```